Meta/check-newlines-at-eof.py

```diff
@@ -0,0 +1,62 @@
+#!/bin/python3
+
+import os
+import subprocess
+import sys
+
+os.chdir(os.path.dirname(__file__) + "/..")
+
+files = subprocess.run(
+    [
+        "git", "ls-files", "--",
+        "*.cpp",
+        "*.h",
+        "*.gml",
+        "*.html",
+        "*.js",
+        "*.css",
+        "*.sh",
+        "*.py",
+        "CMake*.txt",
+        "**/CMake*.txt",
+        ":!:Base",
+        ":!:Kernel/FileSystem/ext2_fs.h",
+        ":!:Libraries/LibC/getopt.cpp",
+        ":!:Libraries/LibCore/puff.h",
+        ":!:Libraries/LibCore/puff.cpp",
+        ":!:Libraries/LibELF/exec_elf.h"
+    ],
+    capture_output=True
+).stdout.decode().strip('\n').split('\n')
+
+MISSING_NEWLINE_AT_EOF_ERRORS = []
+MORE_THAN_ONE_NEWLINE_AT_EOF_ERRORS = []
```

Comment:
Not sure it matters super much for this straight port of the bash script but seeing an all caps variable name that's not a constant in python code is surprising to me. I tend to follow either PEP-8 (with a checker like flake8) or the Google style guide for python files. https://google.github.io/styleguide/pyguide.html#s3.16-naming

Reply:
ok, no problems.

Reply:
> this straight port of the bash script

I wouldn't call it a straight port :^)